Apps/WebClient/src/ClientApp/src/store/modules/credential/types.ts

```diff
@@ -22,34 +21,30 @@ export interface CredentialState {
 export interface CredentialGetters
     extends GetterTree<CredentialState, RootState> {
     connection(state: CredentialState): WalletConnection | undefined;
-    credentials(state: CredentialState): WalletCredential[];
+    credentials(state: CredentialState): WalletCredential[] | null;
 }
 
 type CredentialContext = ActionContext<CredentialState, RootState>;
 export interface CredentialActions
     extends ActionTree<CredentialState, RootState> {
-    retrieveConnection(
+    createConnection(
         context: CredentialContext,
-        params: { hdid: string }
+        params: { hdid: string; targetIds: string[] }
```

Comment:
I think the targetIds parameter needs to be removed.

Reply:
I meant to remove it :)